package.json

```diff
@@ -56,8 +56,8 @@
     "build-storybook:vue": "node --max-old-space-size=8192 node_modules/@storybook/vue/bin/build.js -c .storybook/vue -o storybook-static-vue",
     "ci-check": "yarn wca && yarn format:diff && yarn lint:src && yarn typecheck && yarn build && yarn test:unit && yarn lint:dist",
     "clean": "gulp clean",
-    "format": "prettier --write \"**/*.{css,js,md,scss}\"",
-    "format:diff": "prettier --check \"**/*.{css,js,md,scss}\"",
+    "format": "prettier --write \"**/*.{css,js,ts,md,scss}\"",
+    "format:diff": "prettier --check \"**/*.{css,js,ts,md,scss}\"",
     "format:staged": "prettier --write",
```

Comment:
I have also started to format the .ts.
They were already in lint-staged, but not in the yarn format script.